api/desecapi/tests/test_totp.py

```diff
@@ -124,3 +175,39 @@ def test_workflow(self):
             self.assertResponse(
                 response, status.HTTP_400_BAD_REQUEST, {"code": ["Invalid code."]}
             )
+
+        # Additional token needs unique name
+        response = self.client.post(self.reverse("v1:totp-list"))
+        self.assertResponse(
+            response,
+            status.HTTP_400_BAD_REQUEST,
+            {"non_field_errors": ["The fields user, name must make a unique set."]},
```

Comment:
Change to something more user-friendly

Reply:
done: `An authentication factor with this name already exists.`